rust/worker/Cargo.toml

```diff
@@ -11,6 +11,11 @@ path = "src/bin/query_service.rs"
 name = "compaction_service"
 path = "src/bin/compaction_service.rs"
 
+[[bench]]
+name = "distance_metrics"
+path = "src/benches/distance_metrics.rs"
+harness = false
```

Comment:
learning: what does this do?

Reply:
harness disables libtest benchmarking since I am using the `criterion` crate